src/main/java/com/geekbeast/rhizome/configuration/hazelcast/HazelcastConfiguration.java

```diff
@@ -33,6 +35,7 @@
     private final String              group;
     private final String              password;
     private final int                 port;
+    private final String              role;
```

Comment:
I feel like this makes more sense as a boolean as there only two possible roles.